salt/journal/init.sls

```diff
@@ -74,16 +68,15 @@ assets-nginx-configuration:
         - require:
             - journal-folder
 
-# files and directories must be readable and writable by both elife and www-data
-# they are both in the www-data group, but the g+s flag makes sure that
-# new files and directories created inside have the www-data group
+# files and directories must be writable by www-data,
+# the user the container is running as
 logs-directory:
     file.directory:
         - name: /srv/journal/var/logs
```

Comment:
Probably don't actually need `var` anymore?

Reply:
Just keeping it consistent with the filesystem inside the container to prevent confusion

Reply:
Also migrating the path of a volume on the host is a pain

Reply:
Guess that explains https://github.com/elifesciences/journal-formula/blob/d06cdff4c0919bb302489489ed4fc7ed515d41a9/salt/journal/init.sls#L58-L59

Reply:
At least that's readonly so cleanup is just removing the old file with another state; but `logs/` is writable so can't move it without stopping the containers preemptively, move files, restart.